### ISSUE TITLE
UI improvement 

### ISSUE THREAD

**Issue by Engineervinay** [Engineervinay/search-hackernews#3]
**Any new developer out there learning react.js can create Pull request to improve UI of this Website. I'll guide you if you get any problem.**

**Comment by Aman-Mandal** [comment]
I would like to work on this !! Assign this to me Sir :)

**Comment by Engineervinay** [comment]
@Aman-Mandal feel free to message me if you get any problems.

**Comment by AnnieDebbie** [comment]
Hello Sir,
I would also really like to work on this if there is still an opportunity to
I could also help in updating the README. We can create a separate issue for that if you agree.

**Comment by Engineervinay** [comment]
Yeah sure 👍 you can also work on UI or create a new issue for README.
And please let me know if you are participating in hacktoberfest or not , so that i can add hacktoberfest-accepted tag to your PR. 
Reach out to me if you get any problems

**Comment by AnnieDebbie** [comment]
Thank you and yes I am participating in the hacktoberfest

**Comment by AnnieDebbie** [comment]
Hello, 
I'd like to confirm if you have any preferred commit styles and branch naming styles. Also just how many changesets should a pull request have

**Comment by Engineervinay** [comment]
> Hello, I'd like to confirm if you have any preferred commit styles and branch naming styles. Also just how many changesets should a pull request have

Hey,
Honestly, we don't have any commit styles or naming schemes for branches. and you can put many commits together with no problem, if you know how to squash commits then you can do that but otherwise, it's fine even if there are multiple commits.
You can make changes as you want, just add screenshots in PR. 
All the best 👍

**Comment by AnnieDebbie** [comment]
Hello
I hope you are doing well
I have started working on the UI.
I converted the comments from html to plain text. I can push that and create a PR if you'd like

Also, I reached out to a product designer friend of mine to design the screens which I will then code. I hope you don't mind. 
If you approve, send me your email address so you can be given access to the Figma file. Or, I can send screenshots of the screens.
You can also signify what extra features you'd like for the UI so I can relay that to the designer

**Comment by Engineervinay** [comment]
Hey, @AnnieDebbie I am really sorry for the delay. I will merge this PR and yes you can work on New UI just create a new PR, and ADD screenshots of your newly created UI.

**Comment by AnnieDebbie** [comment]
Hello @Engineervinay 
It's been a minute. I was under the weather. 
Here's a screenshot of the proposed design for the home page.

![Screenshot from 2022-10-28 21-23-54@2x](https://user-images.githubusercontent.com/32017636/198726156-781fdd8f-6963-4331-8899-0ebd8cf94802.png)

**Comment by Engineervinay** [comment]
Hey, we can go with this design but I don't think we need recently visited sites.

**Comment by AnnieDebbie** [comment]
Okay. Got it

**Comment by tikpeme** [comment]
Hey @Engineervinay,

Is this still open for contributions? I would like to try an make a contribution as my first time doing so.

Thanks
